utkdigitalinitiatives/polk-correspondence-app
# Address PTC-130 :rage:

**JIRA Ticket**: [https://jirautk.atlassian.net/browse/PTC-130](https://jirautk.atlassian.net/browse/PTC-130)

# What does this Pull Request do?

Resolves a major issue in my development workflow where the Polk version of the UT Libraries header had diverged from my master at https://github.com/utkdigitalinitiatives/utk_lib_header. This pull request integrates a compiled code that is shipped to the proper directory in the polk-correspondence-app repo. The polk custom subheader is then added in addition to this code and rendered in a new target div with the ID `utk-custom-subheader`.

# What's new?

Removed a lot of redundant code (both JS and CSS) that existed in both repos, leaving only the code in place that is necessary for compile that which is specific to the  Polk app.

Example:
* Removed unneeded JS/CSS.
* Cleanup of Polk specific /src structure to be more modular for similar projects in the future.
* Updated imports for precompiled JS/CSS from the [utk_lib_header](https://github.com/utkdigitalinitiatives/utk_lib_header) repo.
* Updated target div ID to `utk-custom-subheader` for custom subheader.

# How should this be tested?

Describe what steps to take to test this change. 

* `ant` as normal and upload
* Inspect the subheader and note that it renders in `<div id="utk-custom-subheader">...`
* Nothing should actually look different than it previously did within the Polk app.

# Additional Notes:
This is a necessary step forward for testing and pushing both Polk and other projects that will utilize the header. It will allow all changes to made from a single repo in regards to our universal header.

# Interested parties
@markpbaggett @CanOfBees
